src/ngx_http_lua_control.c

```diff
@@ -248,6 +248,8 @@ ngx_http_lua_ngx_redirect(lua_State *L)
                           "the headers");
     }
 
+    len = ngx_http_lua_safe_header_filed_len(p, len);
```

Comment:
I have problem understanding the meaning of the word "filed" used here. Seems like `value` is a better word here for `filed`?

Reply:
sorry, it's a typo, it's `field` since we want to take care of both the header name and the header value.